api/types/src/transaction.rs

```diff
@@ -1228,22 +1269,28 @@ impl TryFrom<SingleKeySignature> for AccountAuthenticator {
     type Error = anyhow::Error;
 
     fn try_from(value: SingleKeySignature) -> Result<Self, Self::Error> {
-        let key = match value.public_key {
-            PublicKey::Ed25519(p) => {
-                let key = p
-                    .inner()
-                    .try_into()
-                    .context("Failed to parse given public_key bytes as Ed25519PublicKey")?;
-                AnyPublicKey::ed25519(key)
-            },
-            PublicKey::Secp256k1Ecdsa(p) => {
-                let key = p
-                    .inner()
-                    .try_into()
-                    .context("Failed to parse given public_key bytes as Secp256k1EcdsaPublicKey")?;
-                AnyPublicKey::secp256k1_ecdsa(key)
-            },
-        };
+        let key =
+            match value.public_key {
```

Comment:
hmm it seems like it's because `match value.public_key {` is now on a newline. When i run `cargo +nightly fmt` it keeps on moving it to a newline 🤷‍♂️